main/guides/getting-started/README.md

```diff
@@ -297,4 +297,17 @@ When the transaction is complete you will notice some IST has been debited from
 
 ![Finished transaction](./assets/new_006.png)
 
-Congratulations! You got your first Agoric dapp up and running!
+Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
+- Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
+```sh
+yarn start:docker
+```
+
+- Deploying a Contract: You have learned the steps to deploy a smart contract onto the Agoric platform!
+```sh
+yarn start:contract
+```
+
+- Making an Offer: Finally, you have learned how to make offers within the dapp and add items to your Keplr wallet!
```

Comment:
I'd rather not say that adding Keplr accounts is a key take-away from our docs. That's something Keplr documents pretty well.

And again, I'm more comfortable saying that they managed to make an offer than that they learned anything about it.

In the video, I'd like us to focus a bit more on the offer step: try to make an offer with <0.25 IST and see what the contract says. Likewise, make an offer with >3 items. And make the point that what the user is signing is a _conditional_ offer. Regardless of how the dapp and the contract behave, the give / want constraints that they sign in Keplr are enforced by the platform.

We could put a bit more of that in the prose. With that, we could also add here that they learned about the conditional nature of offers on the Agoric platform.

Reply:
@dckc:
- Re-worded that part
- Removed code snippets
- Added content in "Making an Offer" section about bidding <0.25 IST or purchasing >3 items
- Will include this in new video / screenshots as well